src/SignaturePad.Forms.Droid/SignaturePad.Forms.Droid.csproj

```diff
@@ -31,8 +31,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.4.0" PrivateAssets="All" />
-    <PackageReference Include="Xamarin.Forms" Version="2.5.0.280555" />
+    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.5.4" PrivateAssets="All" />
+    <PackageReference Include="Xamarin.Forms" Version="3.1.0.469394-pre1" />
```

Comment:
Is it possible to use a stable version of forms here?

Reply:
yes, I am in progress